NEWS.md

```diff
@@ -1,4 +1,8 @@
-# mlr3extralearners 0.6.1-9000
+# mlr3extralearners 0.6.2-9000
```

Comment:
I use the NEWS for 0.6.1-9000 to collect the news for the next release so the dbarts update belongs there

Reply:
Roger that, fixed